### PR TITLE
Sanitized params

### PR DESCRIPTION
**Pull request by dasunhegoda**
Sanitized params in the Odoo sync script

### CODE REVIEW

**Comment by dasunhegoda** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by atelal — APPROVED** [review]

File: scripts/sync_issue_to_odoo.py [hunk 1]
```diff
@@ -12,11 +20,14 @@
 username = os.getenv("ODOO_USERNAME")  # Odoo login username
 password = os.getenv("ODOO_PASSWORD")  # API Key is used
 project_name = "OpenSPP"  # Project name in Odoo
+issue_title = os.getenv("TITLE", "")[:256]
+issue_body = os.getenv("BODY", "")[:256]
+github_issue_id = os.getenv("ISSUE", "")[:256]
```
Comment (atelal) [inline scripts/sync_issue_to_odoo.py]: I think you should be able to do this:
```suggestion
issue_title = sanitize(os.getenv("TITLE", "")[:256])
issue_body = sanitize(os.getenv("BODY", "")[:256])
github_issue_id = sanitize(os.getenv("ISSUE", "")[:256])
```

File: scripts/sync_issue_to_odoo.py [hunk 2]
```diff
@@ -12,11 +20,14 @@
...
 
-# GitHub issue data, passed as script arguments
-issue_title = sys.argv[1]
-issue_body = sys.argv[2]
-github_issue_id = sys.argv[3]  # Unique GitHub Issue ID
+# GitHub issue data, passed as script arguments and sanitized
+issue_title = sanitize(issue_title)
+issue_body = sanitize(issue_body)
+github_issue_id = sanitize(github_issue_id)
```
Comment (atelal) [inline scripts/sync_issue_to_odoo.py]:
```suggestion
```

**Review by atelal — COMMENTED** [review]
Looks good to me otherwise, need to check with @kneckinator

**Review by kneckinator — APPROVED** [review]
Looks good to me

**Review by atelal — APPROVED** [review]

**Pull request merged**